src/app/page.tsx

```diff
@@ -1,8 +1,23 @@
+'use client'
+
 import { Button, Switch } from "@nextui-org/react";
 import { FaCircleInfo } from "react-icons/fa6";
 import RunVocoderForm from "./ui/run-vocoder-form";
+import { initFbase } from "../../firebase/firebaseApp";
+import { getAuth, signInWithPopup, GoogleAuthProvider } from "firebase/auth";
+import { useAuthState } from "react-firebase-hooks/auth";
 
 export default function Home() {
+
+  initFbase();
```

Comment:
What is the point of this function if it just returns app? Can just import app directly if needed